lib/govuk/country_registry.rb

```diff
@@ -0,0 +1,23 @@
+module Govuk
```

Comment:
Our file's called `country_registry.rb` (as is the spec), but our class is called `CountryRegister`.